client/src/components/pages/Home/ReportBuilding/Form/index.js

```diff
@@ -0,0 +1,197 @@
+import React, { Component } from 'react';
+import { Steps } from 'antd';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+
+import FirstStep from './FirstStep';
+import SecondStep from './SecondStep';
+import ThirdStep from './ThirdStep';
+import styles from './form.module.css';
+
+const { Step } = Steps;
+
+const steps = [
+  {
+    title: 'Empty Building',
+  },
+  {
+    title: 'Extra Information',
+  },
+  {
+    title: 'Personal Information',
+  },
+];
+
+class Form extends Component {
+  state = {
+    current: 0,
+    stepOneValues: {
+      city: '',
+      address: '',
+      previousUse: '',
+      owner: '',
+      isOwnerLocal: '',
+    },
+    stepTwoValues: {
+      emptyPeriod: '',
+      extraInfo: '',
+      preferredUse: '',
+      thumbnail: '',
+    },
+    stepThreeValues: {
+      reporterName: '',
+      reporterEmail: '',
+      reporterAddress: '',
+      receiveNotifications: '',
+      shareData: '',
+    },
+  };
+
+  getStepOneValues = values => {
+    const { stepOneValues } = this.state;
+    this.setState({
+      stepOneValues: {
+        ...stepOneValues,
+        ...values,
+      },
+    });
+  };
+
+  getStepTwoValues = values => {
+    const { stepTwoValues } = this.state;
+    this.setState({
+      stepTwoValues: {
+        ...stepTwoValues,
+        ...values,
+      },
+    });
+  };
+
+  getStepThreeValues = values => {
+    const { stepThreeValues } = this.state;
+    this.setState({
+      stepThreeValues: {
+        ...stepThreeValues,
+        ...values,
+      },
+    });
+  };
+
+  handleConfirm = values => {
+    const { stepThreeValues } = this.state;
+    this.setState(
+      {
+        stepThreeValues: {
+          ...stepThreeValues,
+          ...values,
+        },
+      },
+      () => this.sendData(),
+    );
+  };
+
+  sendData = async () => {
+    const { stepOneValues, stepTwoValues, stepThreeValues } = this.state;
+    const data = {
+      ...stepOneValues,
+      ...stepTwoValues,
+      ...stepThreeValues,
+    };
+    console.log('data: ', data);
+    try {
+      const dummy = {
+        city: 'Morecambe',
+        latitude: 52.06835,
+        longitude: -1.86108,
+        address: 'Morecambe',
+        owner: 'Someone',
+        isOwnerLocal: 'Yes',
+        preferredUse: 'N/A',
+        emptyPeriod: '1 year',
+        extraInfo: 'more info about this building',
+        approved: true,
+        receiveNotifications: false,
+        reporterName: 'Som',
+        reporterEmail: 'ahmed@gmail.com',
+        reporterAddress: 'gaza',
+      };
+      await axios.post('/api/v1/report-building', dummy);
```

Comment:
I added dummy object to check Axios request but it gives 400 bad request 
Can you check it ? 
@ahmedisam99